assets/src/blocks/ENForm/FormGenerator.js

```diff
@@ -1,31 +1,32 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Since it seems that this is only for the last block of code (from line 337) maybe you could add it right before that instead of at the top of the file? 🤔 
Alternatively we could rewrite that last block of code to not use `?.` since it seems to be what is causing the issue

Reply:
Done.